client/src/main/java/com/influxdb/client/BucketsApi.java

```diff
@@ -207,6 +207,14 @@ Bucket createBucket(@Nonnull final String name,
     @Nonnull
     List<Bucket> findBucketsByOrgName(@Nullable final String orgName);
 
+    /**
+     * List all buckets.
+     *
+     * @return List all buckets
+     */
+    @Nonnull
+    List<Bucket> findBuckets(BucketsQuery query);
```

Comment:
Please add `@Nonnull` annotation:

```suggestion
    List<Bucket> findBuckets(@Nonnull final BucketsQuery query);
```